docs/Governance.md

```diff
@@ -0,0 +1,223 @@
+arises.
+
+### Regular Maintainers
+
+Regular maintainers are people who have commit access to Homebrew but are not
+expected or obligated to help with legal, technical or administrative issues
+that arise impacting the project itself. Regular maintainers can be members of
+multiple GitHub repositories or may only be maintainers of one repository, such
+as `homebrew/livecheck`.
+
+Regular maintainers spend almost all of their time around Homebrew providing
+code review or submitting pull requests of their own, although they should
+always prioritise the work of others above their own, with rare exceptions such
+as urgent security fixes.
+
+Regular maintainers do not routinely have access to Homebrew's hardware
+_(but can be granted access)_ and are not expected to do sysadmin work. Regular
+maintainers have access to CI for the purposes of build queue control &
+triggering jobs, and have the ability to manually request a rebuild via
+`@BrewTestBot test this please`.
+
+## Accountability
+
+Everyone is accountable for their words & actions, or lack thereof, in Homebrew,
+and nobody has unchecked power or influence. The below list covers various
+situations that may arise and how to handle them.
+
+### The Lead Maintainer Position & The Holder Of That Position
+
+At least once a year, provisionally September 14th, the PLC is mandated to
+affirm or withhold its consent for the lead maintainer role to exist in its
+current form as a distinct entity. The lead maintainer may vote on this issue.
+In the event of a tie regular maintainers are asked to vote on the issue and
+collectively act as the sole tie-breaking vote.
+
+On the same day the PLC is additionally mandated to vote on whether to affirm
+or withhold its consent for the current occupant of the lead maintainer role to
+remain in post. The lead maintainer _may not_ participate in this vote, and
+again in the event of a tie regular maintainers will be asked to provide a
+collective tiebreaker.
+
+Neither of the above bar any member of the PLC calling a vote on either issue
+at an earlier point in time, although doing so should be considered frowned
+upon in all but the most severe of circumstances. In both votes a simple
+majority suffices as a valid determination of the will of the PLC.
+
+If the lead maintainer position becomes vacant for any reason the PLC is
+mandated to fill the vacancy, unless a vote is held to absorb the position.
+Any maintainer or member of the PLC can nominate a new lead maintainer, but
+confirming a new lead maintainer requires every member of the PLC to vote
+and 2/3 of those votes to be in the affirmative.
+
+### Maintainer Pull Requests and Issues
+
+Maintainers may not close other maintainer's Pull Requests or Issues without
+the consent of the maintainer in question. The two exceptions of this rule are:
+
+_Inactivity_: If a Pull Request or Issue is inactive for 28 days, any maintainer
```

Comment:
Doesn’t the `stale-bot` do this already?

Reply:
No, not in some cases. In `brew` for example there's a bot that automatically tags `brew` PRs with the `in progress` label, which stops stale-bot doing its thing.